pennylane/templates/subroutines/qsvt.py

```diff
@@ -507,6 +483,92 @@ class QSVT(Operation):
     >>> q_script = qml.tape.QuantumScript(ops=[qml.QSVT(block_encoding, phase_shifts)])
     >>> print(q_script.expand().draw(decimals=2))
```

Comment:
I tried an alternative way to draw the expanded circuit, but it didn't work:
`print(qml.draw(example_circuit, level=3)())`

Do we think this is an issue with `QSVT` or `qml.draw`?